src/components/PaymentButtons/PaymentButtons.scss

```diff
@@ -57,8 +57,15 @@
             max-height: 1000px;
             width: 100%;
             margin-bottom: 55px;
-            transition: all .3s ease;
+            transition: all .5s ease;
             overflow: hidden;
+            margin-top: 40px;
+
+            @include respond(phone){
+                margin-left: -28%;
+                width: 127%;
```

Comment:
This also looks wrong, both negative margin and width above 100%

Reply:
I got rid of negative margins. I positioned elements differently.